game/factions/faction.py

```diff
@@ -543,6 +543,8 @@ def apply_mod_settings(self, mod_settings: Optional[ModSettings] = None) -> None
             self.remove_preset("LvS-103 Rb103A Mobile")
             self.remove_preset("LvS-103 Rb103B")
             self.remove_preset("LvS-103 Rb103B Mobile")
+        if not mod_settings.coldwarassets:
+            self.remove_aircraft("EA_6B")
```

Comment:
This looks a bit odd... care to explain? 😅

I'm assuming it's a fix for something that was forgotten when introducing support for the EA-6B?
Also, I'm expecting the B_47, Tu-16, etc. in this file 😅

Reply:
Oops! Forgot to do something here 😬

Reply:
This has now been done.